main.go

```diff
@@ -118,6 +118,11 @@ func wrapMain() error {
 			Usage:  "variables to use while templating manifests",
 			EnvVar: "PLUGIN_VARS",
 		},
+		cli.BoolFlag{
+			Name:   "expand-env-vars",
+			Usage:  "expand environment variables oontents on vars",
```

Comment:
typo

Reply:
@fsouza everything looks good if you want to fix this typo.

Reply:
ooops, my bad. will fix.